identity-composer-directive.js

```diff
@@ -34,38 +34,221 @@ function brIdentityComposer($rootScope, brCredentialLibraryService) {
     scope.modal = {show: false};
     scope.loading = false;
 
+    scope.selectedPage = true;
+
+    scope.page = 'front';
+    scope.selectedCredentials = [];
+    scope.allCredentials = [];
+
+    scope.claimsPartiallyFulfillable = false;
```

Comment:
Why the EOLs above? Reading it, I can't understand the purpose of the separate blocks.